iOS/MVC/README.md

```diff
@@ -0,0 +1,46 @@
+# MVC
+
+## Model ViewController
+
+### What is MVC?
```

Comment:
We should remove `#MVC` because it is duplicated with `## Model View Controller`

then, change `## Model View Controller` to `# Model View Controller`

Reply:
#MVC removed